com.unity.render-pipelines.high-definition/Runtime/ShaderLibrary/ShaderVariablesFunctions.hlsl

```diff
@@ -233,4 +233,22 @@ uint2 ComputeFadeMaskSeed(float3 V, uint2 positionSS)
     return fadeMaskSeed;
 }
 
+float4x4 _GetCameraViewProjMatrixRaw()
```

Comment:
Note, you must rely on testing SHADEROPTIONS_CAMERA_RELATIVE_RENDERING for absolute or not.
+ don't use "_" for function, we use it only (for legacy rasons) for constant buffer value.

So here is my suggestion.
remove the change in 
ShaderVariablesFunctions.hlsl

replace the code in PickingSpaceTransforms.hlsl by


// The scene picking pass is not under the control of SRP. The camera matrix is not camera relative
// when using _CameraViewProjMatrix from scene picking pass we must remove the translation to be camera relative
float4x4 ScenePickingGetCameraViewProjMatrix()
{
    float4x4 translationMatrix = {
            { 1.0 ,0.0 , 0.0, -_WorldSpaceCameraPos.x },
            { 0.0 ,1.0 , 0.0, -_WorldSpaceCameraPos.y },
            { 0.0 ,0.0 , 1.0, -_WorldSpaceCameraPos.z },
            { 0.0 ,0.0 , 0.0, 1.0} };

    return mul(_CameraViewProjMatrix, translationMatrix);
}

#if (SHADEROPTIONS_CAMERA_RELATIVE_RENDERING != 0)
#define _CameraViewProjMatrix ScenePickingGetCameraViewProjMatrix()
#endif

And thus we don't need to touch _CameraViewProjMatrix .  (note that _CameraViewProjMatrix  is used in an upcoming VFX PR, so with above suggestion it will still work. We have only issue with scene picking, so let's fix it only for it.

Reply:
Done